client/src/graphs/templates/ui/TemplateMenu.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+  import { graph } from "@graph/global";
+  import CPopover from "@ui/core/Popover.vue";
+  import GWell from "@ui/graph/GWell.vue";
+  import { useTemplate } from "../useTemplate";
+  import GButton from "@ui/graph/button/GButton.vue";
+  import TemplateItem from "./TemplateItem.vue";
+import { computed } from "vue";
+
+  const {
+    templates,
+    userTemplates,
+    addCurrentGraphAsTemplate,
+    loadTemplate,
+    clearUserTemplates,
+    deleteUserTemplate,
+  } = useTemplate(graph.value);
```

Comment:
why is this not useGraphTemplate? I suggest it to be that cus theres something called useTemplateRef and useTemplate in vue and it can be confusing in the context of a vue SFC

Reply:
AND if its useGraphTemplate, you dont have to include the name template in each exported property name